src/app/stores/app_store.ts

```diff
@@ -1214,6 +1214,8 @@ export class AppStore extends BaseStore {
       style: deepClone(Prototypes.PlotSegments.defaultAxisStyle),
       numericalMode: options.numericalMode,
       dataKind: dataExpression.metadata.kind,
+      autoDomainMax: true,
```

Comment:
does this require a version bump?

Reply:
No, there is no new functionality, it's just duplication of UI for existing. Let's increase the version number before deploy the new version.